.env

```diff
@@ -0,0 +1,5 @@
+BD_USER = postgres
```

Comment:
The `.env` is meant to hold sensitive variables. So you should add it gitignore, then create a sample env file with only expected values like
```
DB_HOST=
DB_PORT=
```
You can name it `.env.example`. This is the one you should push to github.